casbin/model/policy.py

```diff
@@ -109,13 +97,12 @@ def update_policies(self, sec, ptype, old_rules, new_rules):
 
     def remove_policy(self, sec, ptype, rule):
         """removes a policy rule from the model."""
-
         if not self.has_policy(sec, ptype, rule):
             return False
 
         self.model[sec][ptype].policy.remove(rule)
 
-        return rule not in self.model[sec][ptype].policy
+        return True
```

Comment:
Try to consider this as an additional check?

Reply:
Yes, I wasn't sure about this change since we might possibly have the rule in duplicate in the list of policies (?).
I'll revert it